### PR TITLE
US114065 Skeleton placeholder

### PR DESCRIPTION
**Pull request by cbelan-d2l**
http://design.d2l/patterns/skeleton-loading/

image

### CODE REVIEW

File: src/components/awards-leaderboard-ui.js [hunk 1]
```diff
@@ -34,7 +34,54 @@ class App extends BaseMixin(LitElement) {
...
+				100% { background-color: var(--d2l-color-sylvite); }
+			}
+			.skeleton-awardRow{
+				display: flex;
+				flex-direction: row;
+				align-items: center;
+				padding: 3px;
+			}
+			.skeleton-awardRank{
+				animation: loadingPulse 1.8s linear infinite;
```
Comment (sgrinwis-d2l) [inline src/components/awards-leaderboard-ui.js]: Oooo Fancy

File: src/components/awards-leaderboard-ui.js [hunk 2]
```diff
@@ -64,6 +111,25 @@ class App extends BaseMixin(LitElement) {
 	}
 
 	render() {
+		if(!this.doneLoading){
+			const numberOfItems = 5;
+			const itemsSkeleton = html`
+				<d2l-list-item>
+					<d2l-list-item-content>
+				<div class="skeleton-awardRow">
```
Comment (sgrinwis-d2l) [inline src/components/awards-leaderboard-ui.js]: Nit: Tabbing here looks really weird.

File: src/components/awards-leaderboard-ui.js [hunk 3]
```diff
@@ -97,7 +162,7 @@ class App extends BaseMixin(LitElement) {
 		const myLeaderboard = await LeaderboardService.getLeaderboard(this.orgUnitId, this.sortByCreditsConfig);
 		console.log(myLeaderboard); // eslint-disable-line no-console
 		this.sortedLeaderboardArray = myLeaderboard.Objects;
-		this.doneLoading = true;
+		//this.doneLoading = true;
```
Comment (sgrinwis-d2l) [inline src/components/awards-leaderboard-ui.js]: Do we still need this commented out code?

Comment (cbelan-d2l) [reply src/components/awards-leaderboard-ui.js]: Nope - Fixed

**Pull request merged**